plugin/pickachu/main.py

```diff
@@ -8,22 +8,23 @@
 #              and inserts valid output to the user's buffer.
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+DEFAULT_APP = vim.eval('g:pickachu_default_app')
+
+
 def MainFunction():
 	# This section is for getting the
 	# arguments from the user's Vim
 	# command.
 	arglength = int(vim.eval('a:0'))
-	CHOOSEN_APP = 'color'
+	choosen_app = DEFAULT_APP
 	CHOOSEN_FORMAT = None
 	if arglength > 0:
-		CHOOSEN_APP = vim.eval('a:1')
+		choosen_app = vim.eval('a:1')
 	if arglength > 1:
 		CHOOSEN_FORMAT = vim.eval('a:2')
-	if not CHOOSEN_APP:
```

Comment:
I'm not sure about this, since I'm not a vim-plugin expert, but at first glance it seems redundant